e107_languages/English/admin/lan_admin.php

```diff
@@ -519,3 +519,6 @@
 define("LAN_LOCK", "Lock");
 define("LAN_SYSTEM_USER", "System user");
 define("LAN_ENGINE", "Engine");
+define("LAN_SOCIAL_POST", "Post to Twitter");
```

Comment:
Since these terms are ONLY used by the social plugin, they should be a social language file. 
e_admin.php may need a language file loaded with it.